Sources/Core/API/Label.swift

```diff
@@ -19,6 +19,8 @@ public final class Label: SceneObject, InstanceHashable, ActionPerformer, Plugga
     public internal(set) var scene: Scene?
     /// A collection of events that can be used to observe the label.
     public private(set) lazy var events = LabelEventCollection(object: self)
+    /// Multiple lines support. Should fit its size
```

Comment:
Maybe we can make this describe the property a bit more? How about something like:
```
Whether the label's text should be wrapped on multiple lines in case it doesn't fit its width.
```